### PR TITLE
feat: remove table information dropdown

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: #2963 

### Description:
- removes table information dropdown
- adds tooltip in the tables sidebar
- adds tooltip in the table sidebar (breadcrumb)

https://www.loom.com/share/309c6d20689b4474938aded5846b5b7a

### CODE REVIEW

**Comment by rephus** [comment]
Should we also add tooltip on hover  after a table is selected ? like here if I hover on customers: 

image

@PriPatel

File: packages/backend/src/services/ProjectService/ProjectService.mock.ts [hunk 1]
```diff
@@ -160,6 +160,9 @@ export const expectedAllExploreSummary: SummaryExplore[] = [
         name: validExplore.name,
         label: validExplore.label,
         tags: validExplore.tags,
+        databaseName: validExplore.tables[validExplore.baseTable].database,
+        schemaName: validExplore.tables[validExplore.baseTable].schema,
```
Comment (rephus) [inline packages/backend/src/services/ProjectService/ProjectService.mock.ts:164]: this is not needed, right ?

Comment (IrakliJani) [reply packages/backend/src/services/ProjectService/ProjectService.mock.ts:164]: @rephus  this is for TableInfo component, which is not being used at this moment. I added a loom where I explain why I kept TableInfo/LineageViews

**Comment by IrakliJani** [comment]
> Should we also add description tooltip on hover after a table is selected ? like here if I hover on customers:
@rephus 

yep that's a good idea

**Review by rephus — APPROVED** [review]
LGTM

**Comment by PriPatel** [comment]
yup, looks good to me

**Pull request merged**